gradio/route_utils.py

```diff
@@ -583,3 +599,64 @@ def starts_with_protocol(string: str) -> bool:
     """
     pattern = r"^[a-zA-Z][a-zA-Z0-9+\-.]*://"
     return re.match(pattern, string) is not None
+
+
+def delete_files_created_by_app(blocks: Blocks, age: int | None) -> None:
```

Comment:
I believe this won't delete uploaded files (which can take up a lot of disk space in viral Spaces

Reply:
Yes good point! Fixed now